### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Jobs_Keyboard: Focus is not visible on Scroll bar.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3614]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.4.7; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS2.4.7

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA application.
3.Tab till View Menu-->Tool Windows-->Azure Explorer and press enter.
4.Tab till AzureHD InsightsLink a cluster and press enter.
5.Enter all the fields and press ok.
6.Tab till the cluster that was linked and press enter.
7.Select “Jobs” link.
8.Verify whether focus is visible on Scroll bar or not.

**Actual Result:**
Focus is not visible on Scroll bar. When arrow keys are used then only user can see that page is moving up and down.

**Expected Result:**
Focus should be visible on Scroll bar.

**User Impact:**
Keyboard users will face difficulty in knowing where the focus is pointing if Focus is not visible on the scroll bar.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

[Bug_Focus is not visible on scroll bar in jobs page.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646708/Bug_Focus.is.not.visible.on.scroll.bar.in.jobs.page.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `scroll` button, we can scroll down or scroll up with keyboard but the focus is invisible.
![image](https://user-images.githubusercontent.com/32627233/65660466-660ba700-e061-11e9-9598-47e15d5d504a.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749714
